connectors/gitlab/element-templates/gitlab-connector.json

```diff
@@ -181,6 +193,73 @@
         "equals": "releases"
       }
     },
+    {
+      "label": "Operation",
+      "id": "operation",
+      "group": "operation",
+      "description": "Choose operation to perform",
+      "type": "Dropdown",
```

Comment:
Nitpick: Consider adding a default value for this dropdown.

For dropdowns with more than one option, OOTB templates usually have a default (usually the first item). Due to a Modeler bug, we shuouldn't apply defaults to dropdowns with only one option like elsewhere in the commit.

Reply:
I thought about adding this, too, but didn't in the end because the existing operation groups within this template also don't have a default value (see for example releases, issues). This appears to also be case for other templates, e.g. https://github.com/camunda/connectors/blob/9c8831200969f597215317a86294a331dabd524b/connectors/asana/element-templates/asana-connector.json#L102 while for example the GitHub connector consistently uses default values.

I can add a default value, just wanted to point this out.

Reply:
Ok! Like you suggested, let's keep this without a default value to keep it consistent.